src/modules/settings/styles/_rsuite.less

```diff
@@ -111,3 +115,9 @@
     width: 100%;
   }
 }
+
+.@{ns}nav-item-content {
+  svg {
+    width: 100% !important;
```

Comment:
does this stretch the images?

Reply:
It doesn't, the svgs just won't will the bounding box otherwise. this fixes it